WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingPaymentMethod/WooShippingPaymentMethodLine.swift

```diff
@@ -0,0 +1,32 @@
+import Networking
+
+/// Represents payment method line in the order details bottom sheet.
+enum WooShippingPaymentMethodLine: Equatable {
+    case add
+    case card(CardPaymentMethodLineViewModel)
+
+    struct CardPaymentMethodLineViewModel: Equatable {
+        let id: Int64
+        let title: String
+        let isEditable: Bool
```

Comment:
Nit: Since we're hiding the payment section for fulfilled shipments, the section should always be editable when it's visible.